dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/src/Microsoft.ApplicationModel.Resources.vcxproj

```diff
@@ -126,6 +126,9 @@
     <ClCompile>
       <PreprocessorDefinitions>_DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
+    <Link>
+      <AdditionalOptions Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">/debug /debugtype:cv,fixup /opt:ref /opt:icf /incremental:no %(AdditionalOptions)</AdditionalOptions>
```

Comment:
Are there individual linker/build settings to enable these, rather than modifying the options settings directly?

Reply:
Turning on no-increment-link and ICF is going to make the developer inner loop more costly.  Can we limit the impact only to release bits? While I agree with Hui that there's danger in letting debug & release have different behaviors, the scans should only be on the code we're releasing.